Components/Core/Spells/Equivalencies.lua

```diff
@@ -504,40 +574,67 @@ TMW.BE = {
 			-289657, -- Holy Word: Concentration
 			 290641, -- Ancestral Gift
 		},
+		ImmuneToSlows = {
+			   1044, -- Blessing of Freedom
+			  46924, -- Bladestorm (fury)
+			  48265, -- Death's Advance
+			  54216, -- Master's Call
+			  87023, -- Cauterize
+			 201447, -- Ride the Wind (windwalker monk pvp talent)
+			 212552, -- Wraith Walk (F/UHDK PVE talent)
+			 216113, -- Way of the Crane
+			 227847, -- Bladestorm (arms)
+			 287081, -- Lichborne (F/UHDK PVP talent)
+		},
+		DispellableEnrage = {
```

Comment:
Not needed - dispellable enrages can already be checked with "Enraged" as a dispel type (in the same way Poison/Magic/Curse/Disease work)